### PR TITLE
simple username / email screen

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
choose paperkey flow to see it in mock

screen shot 2016-02-29 at 1 24 50 pm

### CODE REVIEW

**Comment by MarcoPolo** [comment]
How do I get to the paperkey flow?

**Comment by chrisnojima** [comment]
start with a clean homedir, select ' already on keybase? log in', choose 'Connect your paper key'

File: shared/login/register/username-or-email/index.render.desktop.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// @flow
+import React, {Component} from 'react'
+import {Text, Icon, Input, Button} from '../../../common-adapters'
+import Container from '../../forms/container.desktop'
+import type {Props} from './index.render'
+
+type State = {usernameOrEmail: string}
+
+class Render extends Component<void, Props, State> {
+  state: State;
```
Comment (MarcoPolo) [inline shared/login/register/username-or-email/index.render.desktop.js]: You have to do this even if you defined state above?

Comment (chrisnojima) [inline shared/login/register/username-or-email/index.render.desktop.js]: it complained cause i think the flow defined stuff doesn't do state: State; itself... maybe?

**Comment by MarcoPolo** [comment]
:shipit:

**Pull request merged**